contrib/download-frozen-image-v2.sh

```diff
@@ -11,7 +11,7 @@ set -eo pipefail
 # scope, but may not accept contributions beyond that.
 #
 # For users who have a similar need but require more flexibility/functionality,
-# refer the the discussion on GitHub, which mentions various alternatives that
+# refer the discussion on GitHub, which mentions various alternatives that
```

Comment:
Looks like this one is missing a word; should probably be `refer to the discussion ....`

Reply:
Thanks!  Modified